meta/data.go

```diff
@@ -34,6 +36,15 @@ type Data struct {
 	MaxNodeID       uint64
 	MaxShardGroupID uint64
 	MaxShardID      uint64
+
+	Logger *log.Logger
+}
+
+// NewData returns a new instance of Data.
+func NewData() *Data {
+	return &Data{
+		Logger: log.New(os.Stderr, "[meta] ", log.LstdFlags),
```

Comment:
@jwilder -- are you suggesting going this in the `init()` method of the package?